packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -108,6 +108,14 @@ export class Runtime {
     supportsCodeGuruProfiling: true,
   });
 
+  /**
+   * The Python 3.9 runtime (python3.9)
+   */
+  public static readonly PYTHON_3_9 = new Runtime('python3.9', RuntimeFamily.PYTHON, {
+    supportsInlineCode: true,
```

Comment:
Does it support inline code yet? Usually this support trails behind the actual runtime support.

If so, can you add this to [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda/test/integ.runtime.inlinecode.ts) and run the verification documented at the top of the file?

Reply:
@nija-at  - support for 3.8 was added
and there is an issue that any version would support inline going forward (https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/80)

But I will add this to the checks.

Reply:
@nija-at - added the test and it worked.